jib-maven-plugin/.gitignore

```diff
@@ -9,4 +9,3 @@ out
 /.settings
 /.classpath
 /.project
-log.txt
```

Comment:
Why remove this?

Reply:
Is this file generated by something? Otherwise, it just thought it isn't common to have this in `.gitignore`.

Reply:
Merging in this, but anyone feel free to restore this if needed.